src/fake-data/fake-data-agg.json

```diff
@@ -5,333 +5,405 @@
       "source": "location/primary_location.json",
       "data_type": "Location",
       "data_origin": "Volunteered",
-      "interaction_date": "",
-      "action": "Add primary location",
+      "timestamp": "",
```

Comment:
fake-data-agg.json not used anymore, no ?

Reply:
Non plus besoin de fake-data 👍